src/constants/map.js

```diff
@@ -273,6 +273,14 @@ const map = {
     toolNames: ['CropPage'],
     annotationCheck: null,
   },
+  fileattachment: {
+    icon: 'ic_placeholder_black_24px',
+    iconColor: null,
```

Comment:
After we've made changes to use StrokeColor in the annotation we should switch iconColor, currentPalette and availablePalettes to StrokeColor